sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/implementation/ChangeFeedMode.java

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation.changefeed.implementation;
+
+public enum ChangeFeedMode {
```

Comment:
Because it is applied to the CF Request Options - like with withFullFidelity() it would be relatively easy to also add the Incremental-To-FullFidelity-Point-In-Time. But yes - if we change the model to push this into parameters I would prefer a similar model how you chose in .Net